packages/aws-lc-fips/aws-lc-fips.spec

```
@@ -0,0 +1,77 @@
+Name: %{_cross_os}aws-lc-fips
+Version: 3.0.0
+Release: 1%{?dist}
+Summary: AWS-LC cryptographic library (FIPS)
+License: Apache-2.0 OR ISC OR BSD-3-Clause OR MIT OR CC0-1.0 OR OpenSSL OR SSLeay-standalone
+URL: https://github.com/aws/aws-lc
+
+Source0: https://github.com/aws/aws-lc/archive/AWS-LC-FIPS-%{version}/aws-lc-AWS-LC-FIPS-%{version}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+Requires: %{_cross_os}glibc
+
+%description
+AWS-LC is a general-purpose cryptographic library maintained by the
+AWS Cryptography team for AWS and their customers. It іs based on code
+from the Google BoringSSL project and the OpenSSL project. This version
+includes FIPS support.
```

I prefer this pattern to minimize the amount of words written in spec files that no one will ever read:
```suggestion
%description
%{summary}.
```